src/core/server/elasticsearch/client/configure_client.test.ts

```diff
@@ -195,40 +197,158 @@ describe('configureClient', () => {
       `);
     });
 
-    it('logs each queries if `logQueries` is true', () => {
-      const client = configureClient(
-        createFakeConfig({
-          logQueries: true,
-        }),
-        { logger, scoped: false }
-      );
+    describe('logs each queries if `logQueries` is true', () => {
```

Comment:
Can we add test cases for when body is null or undefined?

Reply:
Added a test with undefined